src/main/java/com/microsoft/azure/cosmosdb/kafka/connect/sink/CosmosDBSinkTask.java

```diff
@@ -35,16 +40,12 @@ public void start(Map<String, String> map) {
         this.settings = new SinkSettings();
         this.settings.populate(map);
 
-        this.client = new CosmosClientBuilder()
-                .endpoint(settings.getEndpoint())
-                .key(settings.getKey())
-                .userAgentSuffix(SettingDefaults.COSMOS_CLIENT_USER_AGENT_SUFFIX+version())
-                .buildClient();
+        this.client = new CosmosClientBuilder().endpoint(settings.getEndpoint()).key(settings.getKey())

```

Comment:
not: i prefer the original syntax with the different parts of the fluid interface / builder on seperate lines. makes it easier to read. 

is this way preferred by a linter?

Reply:
It seems like VSCode auto formation did this. Will go ahead and fix this.